### DISCUSSION TITLE
Run kickstater with docker

### DISCUSSION THREAD

**Discussion by ilbassa** [luyadev/luya#2163] [Questions & Problems] [answered]
Hi, I have downloaded the project, then I have launched:

- docker compose up -d
- cp configs\env.php.dist env.php
- docker-compose exec luya_web setup

Now I expected the site to be reachable at the address localhost, I missed something?

**Comment by ilbassa** [comment, marked as answer]
I have added
ports:
      - "8080:80"
to the "luya_web" section of docker-compose.yml

Now it is reacheable to localhost:8080

**Reply by nadar** [reply]
@ilbassa yes, thats correct. By default its running on port 80, and you can map and expose the port you wish.

**Comment by nadar** [comment]
I have adjusted that default behavior https://github.com/luyadev/luya-kickstarter/commit/995b12e6d8afebe1db8546856885c161a2c7d7d2 and also updated the install guide https://github.com/luyadev/luya/commit/58f6d9f6d98a6ba877f70bad542dcd834206c750
